src/redisearch_rs/build_utils/src/lib.rs

```diff
@@ -60,11 +60,18 @@ fn rerun_if_rust_changes(dir: &Path) -> std::io::Result<()> {
     rerun_if_changes(dir, &["rs"])
 }
 
-/// Emit granular `rerun-if-changed` statements for each files which are
-/// part of the crates listed in `config.parse.include`.
-/// This ensures cbindgen output is regenerated when the underlying Rust code
-/// is updated.
-pub fn rerun_cbinden(config: &cbindgen::Config) -> Result<(), Box<dyn std::error::Error>> {
+/// Run cbindgen using for the calling crate. Will read from the provided config file and output to the
+/// provided output path.
+pub fn run_cbinden(
+    config_path: impl AsRef<Path>,
```

Comment:
I think it'd be OK to canonically expect the file to be located at `cbindgen.toml` in the current dir.

Reply:
fair enough 👍🏻